apps/web/pages/my-account/returns.vue

```diff
@@ -3,7 +3,7 @@
   <h2 class="hidden md:block col-span-3 typography-headline-4 mb-4 font-bold mx-4 capitalize">
     {{ $t('account.ordersAndReturns.returnsHeading') }}
   </h2>
-  <div v-if="!data?.length" class="col-span-3 text-center">
+  <div v-if="!data?.entries.length" class="col-span-3 text-center">
```

Comment:
not working well, if no orders table header should not be visible

Reply:
just missing an v-else